src/nimscripter/vmops.nim

```diff
@@ -1,11 +1,13 @@
 ## This module contains the a template to implement procedures similar to those that are normally in the `nimscript` module
-import std/os
+import std/[os, osproc]
 import nimscripter/expose
 
 proc exec(s: string) =
-  try:
-    discard execShellCmd(s)
-  except: discard
+  if execShellCmd(s) != 0:
+    raise newException(OSError, s)
```

Comment:
Like I said in the issue this will crash the entire program, so need to think about the best way to handle this, I suppose a PR to Nim might be in order so we can implement the Nimscript code. Cause presently we need this on the Nimscript side https://github.com/nim-lang/Nim/blob/version-1-6/lib/system/nimscript.nim#L263-L274

Reply:
I understand if you don't want to introduce this behavior, however, for my purposes, it does it's job. Please see my comment in #4 . If you'd still like me to remove this, I can create a new branch without this commit.